.github/workflows/actions.yml

```diff
@@ -176,6 +176,19 @@ jobs:
       run: |
         kubectl create ns buoyant-cloud
 
+    - name: Cache Docker layers
+      uses: actions/cache@v2
+      with:
+        path: /tmp/.buildx-cache
+        key: ${{ runner.os }}-buildx-${{ github.sha }}
+        restore-keys: |
+          ${{ runner.os }}-buildx-
```

Comment:
i don't think i see the cache getting used, the build is taking 3:58:
https://github.com/BuoyantIO/linkerd-buoyant/runs/5251084572?check_suite_focus=true#step:13:12

compare to that to this `main` merge using `build-push-action`, takes 5 seconds:
https://github.com/BuoyantIO/linkerd-buoyant/runs/4713356902?check_suite_focus=true#step:9:143

i think the issue is that caching `/tmp/.buildx-cache` only works if we're using buildkit and specifying the cache path. recommend using `build-push-action` and `Move cache`, set `push: false`, and then do the `kind load` ourselves.